src/metabase/search/scoring.clj

```diff
@@ -114,23 +106,42 @@
 
 (defn- consecutivity-scorer
   [query-tokens match-tokens]
-  (largest-common-subseq-length
-   matches?
-   ;; See comment on largest-common-subseq-length re. its cache. This is a little conservative, but better to under- than over-estimate
-   (take 30 query-tokens)
-   (take 30 match-tokens)))
+  (/ (largest-common-subseq-length
+      matches?
+      ;; See comment on largest-common-subseq-length re. its cache. This is a little conservative, but better to under- than over-estimate
+      (take 30 query-tokens)
+      (take 30 match-tokens))
+     (count query-tokens)))
```

Comment:
I have a version that DRYs up all the `(/ foo (count bar))` repetition in my git stash, but it felt like DRYness at the expense of clarity so I rolled with the obvious approach shown here.